File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/dto/request/CancelBatchOrdersReq.java
package com.bitget.openapi.dto.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CancelBatchOrdersReq implements Serializable {

    private static final long serialVersionUID = 2194353520856678089L;

    /**
     * Order Ids
     */
    private List<String> ids;
    /**
     * Currency pair
     */
    private String symbol;
}

File: bitget-golang-sdk-api/pkg/model/spot/order/cancelbatchordersreq.go
package order

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: Batch cancellation request
 */
type CancelBatchOrdersReq struct {

	/**
	 *  Order ids
	 */
	OrderIds []string `json:"orderIds"`
	/**
	 * Currency pair
	 */
	Symbol string `json:"symbol"`
}
